docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -77,6 +80,20 @@ The UI for transforms also has an update that now allows you to move transformat
 
 You can now add override rules that use a regex matcher and overrides that match on field type.
 
+## Query inspector in Explore
+
+You can enjoy all the details query inspector gave you in dashboards now in Explore as well. You can open query inspector tab with the button next to query history. See [Query inspector in Explore]({{< relref "../features/explore/index.md#query-inspector" >}}) for more details.
+
+## $__rate_interval for Prometheus
+
+You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus data source]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.
+
+## Loki toggling parsed fields
```

Comment:
```suggestion
## Loki now toggles parsed fields
```

Reply:
It's not Loki toggling the fields. The user can choose which parsed fields should be shown. When a user toggles a parsed field so it's shown, the other fields are being hidden.

Reply:
@oddlittlebird can you help me find a better title for this feature?

Reply:
Toggle parsed Loki fields?

Reply:
Sounds good to me. Let me change that.